setup.py

```diff
@@ -89,7 +89,7 @@ def get_extension():
         libraries = ('pcap',)
     elif lib_file == "wpcap.lib":
         libraries = ('wpcap', 'iphlpapi')
-        extra_compile_args = ['-DWIN32', '-DWPCAP']
+        extra_compile_args = ['-DWIN32', '-DWPCAP', '-D_CRT_SECURE_NO_WARNINGS']
```

Comment:
It removes the warnings about sscanf, strncpy and fopen functions in pcap_ex.c. It suggests to use Windows-specific functions, but obviously it's not an option. The best solution would be getting rid of pcap_ex.c all together, but it's out of scope of the task.